src/lib/raycaster.js

```diff
@@ -127,9 +127,15 @@ export function initRaycaster(inspector) {
     el: mouseCursor,
     enable: () => {
       mouseCursor.setAttribute('raycaster', 'enabled', true);
+      inspector.container.addEventListener('mousedown', onMouseDown);
+      inspector.container.addEventListener('mouseup', onMouseUp);
+      inspector.container.addEventListener('dblclick', onDoubleClick);
     },
     disable: () => {
       mouseCursor.setAttribute('raycaster', 'enabled', false);
+      inspector.container.removeEventListener('mousedown', onMouseDown);
+      inspector.container.removeEventListener('mouseup', onMouseUp);
+      inspector.container.removeEventListener('dblclick', onDoubleClick);
```

Comment:
As a reminder, you have the dblclick listener commented on your project, so you don't want to add it back here, comment it here as well.

Reply:
@vincentfretin Thanks! I didn't pay attention to that.